Makefile

```diff
@@ -51,8 +51,12 @@ sd-svs-disp: prep-salt ## Provisions SD Submission Viewing VM
 	sudo qubesctl top.enable sd-svs-disp
 	sudo qubesctl --targets sd-svs-disp state.highstate
 
+clean-salt: assert-dom0 ## Purges SD Salt configuration from dom0
+	sudo rm -rf /srv/salt/sd
+	sudo find /srv/salt -maxdepth 1 -type f -iname 'sd*' -delete
+	sudo find /srv/salt/_tops -type l -delete
```

Comment:
This seems a little greedy... on my system, there's a non-securedrop file in that directory (`topd.top`). Perhaps adding a `-iname 'sd*'` would be appropriate here.